src/docs/uxd/alertsNotifications/DOC.md

```diff
@@ -1,5 +1,121 @@
+
+**A process that takes time, which does not need to complete before proceeding (asynchronous)**
+Close modal after confirmation screen, and show inline notification that informs the user that their process has been started or queued (see below).
+After the process is completed, show a toast notification (see below).
+If failure, see error state below.
+
+### Error state for wizard
+
+Communicate errors to the user as soon as possible in a wizard. Use field errors and inline errors as the user goes through the process.
+At the end of a wizard, there may be errors or failures of the entire process. In this case, the user stays in the modal and an inline error state is shown along with the content from the confirmation step.
+The error state should clearly describe what a user needs to do to remediate the error, if anything.
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/inlineErrorWizard.png?raw=true "inlineErrorWizard")
+*Showing the positioning of the inline error in the wizard modal*
+
+## Inline notification
+
+Inline notifications push content down the screen. They should be used to indicate the success of all processes that happen in a modal or wizard.
+These should be dismissible.
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/successMessage.png?raw=true "successMessage")
+*Showing location of success message*
+
+If a process is happening in the background, the success message should indicate that the launching of the process succeeded (not the process itself).
+E.g. "Your request is processing and will complete within 1 hour"
+At the completion of the background process, the user should get a toast notification.
+
+## Toast notifications
+
+Use toast notifications for asynchronous alerts.
+for example, when a background process has completed, show a toast notification.
+Because these notifications are asynchronous to the process, a good notification should include:
+
+- Clear information about who or what prompted the process (if it was not the current user)
+- Clear indication of what process has completed (or failed if an error)
+- A link to see the results of the process (or fix an error)
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/toastNotification.png?raw=true "toastNotification")
+A toast notification can also be an error. In this case it should clearly explain to the user how to solve the problem. Avoid technical error messages.
+
+Toasts should also be used when an action has navigated the user to a new page.
+E.g. Creating a user creates a new portfolio from a landing page. After creating, the user is automatically taken to the new portfolio. In this case, the action was launched from a different page than the user is currently on, so an inline notification is inappropriate. Use a toast notification.
+[Need further examples?]
+
+**Discussion notes**
```

Comment:
Don't need any of this! As a rule, you can ignore anything under "discussion notes" as its just notes on what we talked about. For the final stuff, people don't need to see it